docs/cmd/kn_source_ping_create.md

```diff
@@ -18,8 +18,8 @@ kn source ping create NAME --sink SINK
 
 ```
       --ce-override stringArray   Cloud Event overrides to apply before sending event to sink. Example: '--ce-override key=value' You may be provide this flag multiple times. To unset, append "-" to the key (e.g. --ce-override key-).
-  -d, --data string               Json data to send
-  -e, --encoding string           Preferred encoding format. Valid values: text/base64
+  -d, --data string               Json data to send. Can determine if data is plain text or base64 encoded
```

Comment:
`Json data to send. Can determine if data is plain text or base64 encoded`

Maybe I'd rephrase the whole to read a bit better, but not sure how exactly. I'll try to put some thoughts to it. :)

Reply:
I could think of this:

`Json data to send.This flag can implicitly determine the encoding of the supplied data (text | base64)`